### DISCUSSION TITLE
wslg in windows 10 21h2 howto

### DISCUSSION THREAD

**Discussion by maku** [microsoft/wslg#725] [Q&A]
I found several internet articles saying that wslg is available in "windows 10 21h2".

But his seems not to be true, right? I am in a corporate environment where upgrading to win 11 is not an option.

What do you suggest in such a scenario?

**Comment by Biswa96** [comment]
WSLg is not  available for Windows 10 _officially_ as of this writing. I have some _homemade contraption_ to run WSLg in Windows 10 but it's for personal test only.
